CMakePresets.json

```diff
@@ -0,0 +1,41 @@
+{
+  "version": 2,
+  "cmakeMinimumRequired": {
+    "major": 3,
+    "minor": 19,
+    "patch": 1
+  },
+  "configurePresets": [
+    {
+      "name": "WindowsEditor",
+      "displayName": "Windows Editor Default Configuration",
+      "description": "Editor, asset processor and automated tests.",
+      "generator": "Visual Studio 16 2019",
+      "binaryDir": "${sourceDir}/build/WindowsEditor",
```

Comment:
`${sourceDir}/build/windows_vs2019` matches with what the ci_build scripts use (the ones that Jenkins use) and matches with the documentation. Would be easier for other developers to find their binaries if we keep the same naming.